parlai/core/torch_generator_agent.py

```diff
@@ -715,6 +715,12 @@ def _encoder_input(self, batch):
         """
         return self._model_input(batch)
 
+    def custom_loss_metrics(self, batch, loss_per_token):
```

Comment:
super nit: can we call this something more related to what it's doing?

e.g., `compute_per_token_metrics`?